src/transform/tests/test_runner.rs

```diff
@@ -50,14 +87,26 @@ mod tests {
             )?;
         }
 
-        match test_type {
+        let format_type = if let Some(format) = args.get("format") {
```

Comment:
nit: Since we use the argument keys multiple times, we might want to consider moving them to constants (with the added benefit of documenting which keys exist in a single grouped place).

Reply:
Done.